static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -522,14 +520,13 @@ const FileName = styled('span')`
 
 const RowHeader = styled('span')<{
   expandable: boolean;
-  expanded: boolean;
   isInAppFrame: boolean;
   isSubFrame: boolean;
 }>`
   position: relative;
   display: grid;
-  grid-template-columns: repeat(2, auto) 1fr repeat(2, auto) ${space(2)};
-  grid-template-rows: repeat(2, auto);
+  grid-template-columns: auto 150px 120px 4fr repeat(3, auto) ${space(2)}; /* Adjusted to account for the extra element */
+  grid-template-rows: 1fr; /* Ensures a single row */
```

Comment:
I suspect that the grid columns will depend on the values of `expandable` and `hiddenFrameCount`. I will find some sample data to look at before I can know what to do here.

Reply:
Thanks for the feedback, Once we have clarity from the data, I can help refine the grid logic further. Let me know your findings!

Just to make sure, I am on the right path here, The button (show/hide count) doesn't add a data column but instead adds an extra row header without a corresponding data column. This mismatch causes layout issues like pushing the ExpandCell to the next line.

Reply:
It looks like it's supposed to go into two rows on purpose when the window is narrow. But the colunm widths in that case are kind of broken right now: 

<img width="712" alt="SCR-20241231-ktft" src="https://github.com/user-attachments/assets/2c4b65d1-a4af-407c-9048-f093426e71a5" />

Can you just re-test with a normal-width browser, and a narrow browser window, and share screenshots of the results.

I think you'll want to have something like `grid-template-rows: repeat(2, auto);` with appropriate widths defined for the narrow @media breakpoint.